packages/perseus/src/components/sortable.tsx

```diff
@@ -642,97 +645,64 @@ class Sortable extends React.Component<SortableProps, SortableState> {
         const syncHeight =
             this.props.constraints?.height || layout === Layout.HORIZONTAL;
 
-        _.each(
-            this.state.items,
-            function (item, i, items) {
-                const isLast = i === items.length - 1;
-                const isStatic =
-                    item.state === ItemState.STATIC ||
-                    item.state === ItemState.DISABLED;
-                let margin;
-
-                // @ts-expect-error [FEI-5003] - TS2683 - 'this' implicitly has type 'any' because it does not have a type annotation.
-                if (this.props.layout === Layout.HORIZONTAL) {
-                    // @ts-expect-error [FEI-5003] - TS2683 - 'this' implicitly has type 'any' because it does not have a type annotation.
-                    margin = "0 " + this.props.margin + "px 0 0"; // right
-                    // @ts-expect-error [FEI-5003] - TS2683 - 'this' implicitly has type 'any' because it does not have a type annotation.
-                } else if (this.props.layout === Layout.VERTICAL) {
-                    // @ts-expect-error [FEI-5003] - TS2683 - 'this' implicitly has type 'any' because it does not have a type annotation.
-                    margin = "0 0 " + this.props.margin + "px 0"; // bottom
-                }
+        this.state.items.forEach((item, i, items) => {
+            const isLast = i === items.length - 1;
+            const isStatic =
+                item.state === ItemState.STATIC ||
+                item.state === ItemState.DISABLED;
+            let margin;
+
+            if (this.props.layout === Layout.HORIZONTAL) {
+                margin = "0 " + this.props.margin + "px 0 0"; // right
+            } else if (this.props.layout === Layout.VERTICAL) {
+                margin = "0 0 " + this.props.margin + "px 0"; // bottom
+            }
 
+            cards.push(
+                <Draggable
+                    content={item.option}
+                    key={item.key}
+                    state={item.state}
+                    ref={String(item.key)}
+                    width={syncWidth ? item.width : undefined}
+                    height={syncHeight ? item.height : undefined}
+                    layout={layout}
+                    includePadding={this.props.padding}
+                    margin={isLast && isStatic ? 0 : margin}
+                    endPosition={item.endPosition}
+                    linterContext={PerseusLinter.pushContextStack(
+                        this.props.linterContext,
+                        "sortable",
+                    )}
+                    onRender={this.remeasureItems}
+                    onMouseDown={() => this.onMouseDown(item.key)}
+                    onMouseMove={() => this.onMouseMove(item.key)}
+                    onMouseUp={() => this.onMouseUp(item.key)}
+                    onAnimationEnd={() => this.onAnimationEnd(item.key)}
```

Comment:
`Draggable` doesn't actually have any `onTouch*` props even though we were setting them.

Also, I changed all of the `bind` calls that were here to arrow functions.  The performance characteristics are the same.  This should hopefully be less confusing.